app/routes/index.js

```diff
@@ -107,6 +107,12 @@ const getModel = (data, question) => {
 
 const getHandler = (question) => {
   return (request, h) => {
+    if (question.maybeEligibleContent) {
+      const { url, backLink, nextUrl, maybeEligibleContent } = question
```

Comment:
please create a new function like showNextPage we have for getPostHandler , and put all of thses codes inside of that function . what I mean I want to see something like this when I read the code :
const getHandler = (question) => { return (request,h) => {return  showPageGetRequest(request,h,question)}
and you can write everything in that function.